pkg/util/kubectl/kubectl.go

```diff
@@ -33,11 +33,11 @@ func BinaryForClusterVersion(version *semver.Version) (string, error) {
 
 	switch version.Minor() {
 	case 20:
-		binary = "kubectl-1.20"
+		binary = "kubectl-1.21"
 	case 21:
-		binary = "kubectl-1.22"
+		binary = "kubectl-1.23"
```

Comment:
The 1.21 clusters should use `kubectl-1.21`.

Reply:
How do you spot these typos? Is your mother an eagle? Mine is a mole.